Source/WebCore/rendering/RenderBlock.cpp

```diff
@@ -3108,6 +3108,24 @@ ASCIILiteral RenderBlock::renderName() const
     return "RenderBlock"_s;
 }
 
+String RenderBlock::debugDescription() const
+{
+    if (isViewTransitionPseudo()) {
+        StringBuilder builder;
+
+        builder.append(renderName(), " 0x"_s, hex(reinterpret_cast<uintptr_t>(this), Lowercase));
+
+        builder.append(" ::view-transition"_s);
+        if (style().pseudoElementType() != PseudoId::ViewTransition) {
+            builder.append("-"_s, style().pseudoElementType() == PseudoId::ViewTransitionGroup ? "group("_s : "image-pair("_s);
```

Comment:
Does this output the correct description for view-transition-new/old ?

Fwiw, PseudoId has a TextStream defined which can help with this, alternatively there is also: https://searchfox.org/wubkat/source/Source/WebCore/animation/WebAnimationUtilities.cpp#312 (which could be moved to PseudoElementIdentifier.h)

Reply:
Ah I didn't know about the `TextStream` operator, missed it since it doesn't have `::`.

The WebAnimationUtilities.cpp one would be useful for sure.

This function is mostly the same as the one in `RenderViewTransitionCapture`, which handles the new/old pseudos.